assets/js/show-task.js

```diff
@@ -0,0 +1,49 @@
+function showTask(inputValue, formattedDate, taskList, form) {
+  const task = document.createElement("div");
+  task.classList.add("task");
+
+  const task_content = document.createElement("div");
+  task_content.classList.add("content");
+
+  const inputElement = document.createElement("p");
+  inputElement.classList.add("text");
+  inputElement.innerText = inputValue.value;
+  inputElement.contentEditable = false;
+
+  task_content.appendChild(inputElement);
+
+  const tools = document.createElement("div");
+  tools.classList.add("actions");
+
+  const date = document.createElement("p");
+  date.innerHTML = "Created At: " + formattedDate;
+
+  const done = document.createElement("button");
+  done.classList.add("done");
+  done.innerHTML = `<i class="fa-solid fa-check" id="done"></i>`;
+  done.setAttribute("aria-label", "Mark task as done");
+
+  const edit = document.createElement("button");
+  edit.classList.add("edit");
+  edit.innerHTML = `<i class="fa-solid fa-pen" id="edit"></i>`;
+  edit.setAttribute("aria-label", "Edit task");
+
+  const dlt = document.createElement("button");
+  dlt.classList.add("delete");
+  dlt.innerHTML = `<i class="fa-solid fa-trash-can" id="delete"></i>`;
+  dlt.setAttribute("aria-label", "Delete task");
```

Comment:
We can create a function named `createButton` that'll take `classList`, `icon_type` and `aria_label` as parameters and return the button. 

```js
const editButton = createButton('edit', EDIT_ICON, 'Delete Task')
```

Also keep all the icons in one file in a object or a map and return html from that

```js
const EDIT_ICON = icons[EDIT]
const editButton = createButton('edit', EDIT_ICON, 'Delete Task')
```